fuzz/Cargo.toml

```diff
@@ -35,6 +35,7 @@ path = ".."
 
 [patch.crates-io]
 kvm-bindings = { git = "https://github.com/cloud-hypervisor/kvm-bindings", branch = "ch-v0.6.0-tdx" }
+kvm-ioctls = { git = "https://github.com/rust-vmm/kvm-ioctls", rev = "23a3bb045a467e60bb00328a0b13cea13b5815d0" }
```

Comment:
The main workspace has it like this:

`kvm-ioctls = { git = "https://github.com/rust-vmm/kvm-ioctls", branch = "main" }`

Reply:
I think we should tackle this as part of our bulk upgrade in #6072

Reply:
Okay, so we close this?

Reply:
Yes, lets do that - thanks for bringing it to our attention. To check that I remember to update this when do the big version update.